package.json

```diff
@@ -7,7 +7,7 @@
     "repl": "dotenv -e .env.development -- ts-node scripts/dev-repl.ts",
     "dev": "dotenv -e .env.development -- ts-node-dev --exit-child src/index.ts",
     "start": "dotenv -e .env.development -- ts-node src/index.ts",
-    "test": "dotenv -e .env.test -- prisma migrate reset --force --skip-seed && dotenv -e .env.test -- jest --config jest.config.ts ./test --detectOpenHandles",
+    "test": "docker-compose up -d test_db && sleep 1 && dotenv -e .env.test -- prisma migrate reset --force --skip-seed && dotenv -e .env.test -- jest --config jest.config.ts ./test --detectOpenHandles && docker-compose stop test_db && docker-compose rm -f test_db",
```

Comment:
Might be worth separating this out to its own `scripts/test.sh` file, just for legibility's sake. Would also make debugging easier as with `&&` each command is dependent on the previous command exiting with status code 0. A separate script would let us error out more verbosely.

We could also change the `sleep 1` into a looping liveness probe so we can just run the tests whenever the db is ready.